assets/prefabs/items/WoodenCup.prefab

```diff
@@ -9,6 +9,7 @@
   },
   "FluidContainerItem": {
     "maxVolume": 300,
+    "fillingAmount": 100,
```

Comment:
I think a cup could be filled with one go (and the bucket already demonstrates the subsequent filling):
```suggestion
    "fillingAmount": 300,
```

Reply:
Yeah sure, I know, that was just because it is easier to test it for the cup since drinking is enabled from cup.  I planned to change it later all along.